js/test/components/operatorSignIn/attestation.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { Attestation } from "../../../components/operatorSignIn/attestation";
+import { render } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+jest.mock("../../../hooks/useEmployees.ts", () => ({
+  __esModule: true,
+  findEmployeeByBadge: jest.fn((_employees, badge) => {
+    if (badge === "123") {
+      return {
+        first_name: "Firsty",
+        preferred_first: "Preferredy",
+        last_name: "Lasty",
+        badge: "123",
+      };
```

Comment:
**suggestion (non-blocking):** would it be possible to pull in [Fishery](https://github.com/thoughtbot/fishery) real quick to get us some test factories?

Reply:
Oh, thanks for the reminder!

Reply:
750cf7f267a06a2d57e73ddebcc0a227eb814e93